packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -77,6 +77,28 @@ const AlarmColPool = connectStore(() => ({
   return <Link to={`pools/${pool.id}`}>{pool.name_label}</Link>
 })
 
+const DUPLICATE_MAC_ADDRESS = [
+  {
+    name: _('vifMacLabel'),
+    itemRenderer: vif => vif.MAC,
+    sortCriteria: vif => vif.MAC,
+  },
+  {
+    name: _('vm'),
+    itemRenderer: vif => (
+      <Link to={`vms/${vif.$VM}`}>
+        <Vm id={vif.$VM} />
+      </Link>
+    ),
+    sortCriteria: vif => vif.$VM,
```

Comment:
An ID isn't a good sorting criteria since the users can't see them and they're not human readable. A `name_label` is usually what you want to use as a `sortCriteria`.